src/components/search/results/mobile-partial-results.vue

```diff
@@ -0,0 +1,73 @@
+<template>
```

Comment:
Is it worth it to have separate files for desktop and mobile? The only major difference is the grid instead of the sliding panel (the other ones are two classes and a prop).
I like the implementation, but having them both in the same file is not too convoluted and is convenient if we have to make changes (we won't have to make them twice).

Reply:
I'm not sure about that, I'd rather have two separate files than one and having to check what is applying on mobile or desktop. I think the agreement is to have as less logic as possible in the archetype and having two different files is better for that.

Reply:
yep, @tajespasarela  said that is better to have 2 files instead of having one and play with props, slots and other things